torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -111,6 +111,10 @@ class ProcessGroup {
       std::vector<at::Tensor>& data,
       const AllreduceOptions& opts = AllreduceOptions()) = 0;
 
+  virtual std::shared_ptr<ProcessGroup::Work> allreduce_coalesced(
+      std::vector<at::Tensor>& tensors,
```

Comment:
nit: I believe this vector can be non-const.

Reply:
sorry I don't fully understand, would you mind elaborating?

did you mean that it could be made `const`? It looks like that could be possible since `copy_` is a `const` method